.ci/jenkins/jenkins-trigger-nnp.groovy

```diff
@@ -0,0 +1,31 @@
+String  PR_URL = CHANGE_URL
+String  PR_COMMIT_AUTHOR = CHANGE_AUTHOR
+String  PR_TARGET = CHANGE_TARGET
+String  JENKINS_BRANCH = "kabhira/update-ng-tf-envvars"
+Integer TIMEOUTTIME = "7200"
+
+// Constants
+JENKINS_DIR = '.'
+
+timestamps {
+    node("trigger") {
+
+        deleteDir()  // Clear the workspace before starting
+        // Clone the cje-algo directory which contains our Jenkins groovy scripts
+        try {
+            sh "git clone -b $JENKINS_BRANCH https://gitlab.devtools.intel.com/AIPG/AlgoVal/cje-algo.git ."
+        } catch (e) {
```

Comment:
Can we use the test/ci/jenkins folder to host this script?

Reply:
Done. Thanks for the suggestion. I wasn't aware the "test" folder was already used for storing buildkite scripts.